### PR TITLE
GID-8 Set amount of days to commit

### PR DESCRIPTION
**Pull request by SStefanovskyi**
Set amount of days to commit for TestUser1101.

### CODE REVIEW

File: src/main/java/com/stefanovskyi/gitdeceiver/GitAuthData.java [hunk 1]
```diff
@@ -6,13 +6,15 @@
     private String userEmail;
     private String login;
     private String password;
+    private int amountOfDays;
```
Comment (stefanovskyi) [inline src/main/java/com/stefanovskyi/gitdeceiver/GitAuthData.java]: The `amountOfDays` is not a part of `AuthData`. Please remove it out of here.

File: src/main/java/com/stefanovskyi/gitdeceiver/GitAuthData.java [hunk 2]
```diff
@@ -6,13 +6,15 @@
...
 
     public GitAuthData(String... args) {
         repositoryName = args[0];
         userFullName = args[1];
         userEmail = args[2];
         login = args[3];
         password = args[4];
+        amountOfDays = Integer.parseInt(args[5]);
```
Comment (stefanovskyi) [inline src/main/java/com/stefanovskyi/gitdeceiver/GitAuthData.java]: The `amountOfDays` is not a part of `AuthData`. Please remove it out of here.

File: src/main/java/com/stefanovskyi/gitdeceiver/GitAuthData.java [hunk 3]
```diff
@@ -35,4 +37,8 @@ public String getPassword() {
         return password;
     }
 
+    public int getAmountOfDays() {
+        return amountOfDays;
```
Comment (stefanovskyi) [inline src/main/java/com/stefanovskyi/gitdeceiver/GitAuthData.java]: The `amountOfDays` is not a part of `AuthData`. Please remove it out of here.

File: src/main/java/com/stefanovskyi/gitdeceiver/GitDeceiver.java [hunk 4]
```diff
@@ -13,11 +14,9 @@ public static void main(String args[]) {
 
         fakeRepository.cloneRepository(gitAuthData.getRepositoryName());
         fakeRepository.addRemote(gitAuthData.getRepositoryName());
-        LocalDateTime commitDate = Util.getStartDate(args[5]);
+        LocalDateTime commitDate = Util.getStartDate(args[6]);
```
Comment (stefanovskyi) [inline src/main/java/com/stefanovskyi/gitdeceiver/GitDeceiver.java]: Why did you change parameters ordered number? It will broke workflow for all user that uses previous variant. Please change it back.

File: src/main/java/com/stefanovskyi/gitdeceiver/GitDeceiver.java [hunk 5]
```diff
@@ -3,6 +3,7 @@
 import com.stefanovskyi.gitdeceiver.util.Util;
 
 import java.time.LocalDateTime;
+import java.util.Scanner;
```
Comment (stefanovskyi) [inline src/main/java/com/stefanovskyi/gitdeceiver/GitDeceiver.java]: No usage of Scanner was found in the code. Please remove unused imports.

File: src/main/java/com/stefanovskyi/gitdeceiver/GitDeceiver.java [hunk 6]
```diff
@@ -13,11 +14,9 @@ public static void main(String args[]) {
 
         fakeRepository.cloneRepository(gitAuthData.getRepositoryName());
         fakeRepository.addRemote(gitAuthData.getRepositoryName());
-        LocalDateTime commitDate = Util.getStartDate(args[5]);
+        LocalDateTime commitDate = Util.getStartDate(args[6]);
 
-        int amountOfDays = 333;
-
-        fakeRepository.generateCommits(gitAuthData, fakeRepository, commitDate, amountOfDays);
+        fakeRepository.generateCommits(gitAuthData, fakeRepository, commitDate);
```
Comment (stefanovskyi) [inline src/main/java/com/stefanovskyi/gitdeceiver/GitDeceiver.java]: Just read it from the input and path to commit generator:

```int amountOfDays = Integer.parseInt(args[6]);```
```fakeRepository.generateCommits(gitAuthData, fakeRepository, commitDate, amountOfDays);```

File: src/main/java/com/stefanovskyi/gitdeceiver/FakeRepository.java [hunk 7]
```diff
@@ -47,10 +47,10 @@ public void addRemote(String remoteURI) {
         }
     }
     public void generateCommits(GitAuthData gitAuthData, FakeRepository fakeRepository,
-                                        LocalDateTime commitDate, int amountOfDays) {
+                                        LocalDateTime commitDate) {
```
Comment (stefanovskyi) [inline src/main/java/com/stefanovskyi/gitdeceiver/FakeRepository.java]: No need to remove `amountOfDays`, it still should be passed to method.

**Review by stefanovskyi — CHANGES_REQUESTED** [review]
It was easy task, you just have to add reading `amountOfDays` from params and that's all.

File: src/main/java/com/stefanovskyi/gitdeceiver/GitAuthData.java (hunk 1 above)
Comment (SStefanovskyi) [reply src/main/java/com/stefanovskyi/gitdeceiver/GitAuthData.java]: Done

File: src/main/java/com/stefanovskyi/gitdeceiver/GitAuthData.java (hunk 2 above)
Comment (SStefanovskyi) [reply src/main/java/com/stefanovskyi/gitdeceiver/GitAuthData.java]: Done

File: src/main/java/com/stefanovskyi/gitdeceiver/GitAuthData.java (hunk 3 above)
Comment (SStefanovskyi) [reply src/main/java/com/stefanovskyi/gitdeceiver/GitAuthData.java]: Done

File: src/main/java/com/stefanovskyi/gitdeceiver/GitDeceiver.java (hunk 4 above)
Comment (SStefanovskyi) [reply src/main/java/com/stefanovskyi/gitdeceiver/GitDeceiver.java]: Done

File: src/main/java/com/stefanovskyi/gitdeceiver/GitDeceiver.java (hunk 5 above)
Comment (SStefanovskyi) [reply src/main/java/com/stefanovskyi/gitdeceiver/GitDeceiver.java]: Done

File: src/main/java/com/stefanovskyi/gitdeceiver/GitDeceiver.java (hunk 6 above)
Comment (SStefanovskyi) [reply src/main/java/com/stefanovskyi/gitdeceiver/GitDeceiver.java]: Done

File: src/main/java/com/stefanovskyi/gitdeceiver/FakeRepository.java (hunk 7 above)
Comment (SStefanovskyi) [reply src/main/java/com/stefanovskyi/gitdeceiver/FakeRepository.java]: Done

File: .idea/workspace.xml [hunk 8]
```diff
@@ -0,0 +1,671 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (stefanovskyi) [inline .idea/workspace.xml]: Exclude idea file from git

**Review by stefanovskyi — CHANGES_REQUESTED** [review]

File: .idea/workspace.xml (hunk 8 above)
Comment (SStefanovskyi) [reply .idea/workspace.xml]: Done

**Review by stefanovskyi — APPROVED** [review]

**Comment by stefanovskyi** [comment]
Well done
